docker/Dockerfile.api

```diff
@@ -2,6 +2,14 @@ ARG base_image_tag
 
 FROM deepset/haystack:${base_image_tag}
 
+ENV SERVICE_NAME="gunicorn-service"
+
+RUN addgroup --gid 1001 $SERVICE_NAME && \
+    adduser --gid 1001 --uid 1001 --shell /bin/false --disabled-password $SERVICE_NAME && \
+    mkdir -p /var/log/$SERVICE_NAME && \
+    chown $SERVICE_NAME:$SERVICE_NAME /var/log/$SERVICE_NAME
```

Comment:
Do we need this directory `/var/log/$SERVICE_NAME`? Are we writing any logs there with gunicorn?

Reply:
Good point @bogdankostic I guess this is something we have to fix. Do we need logs? If so, we need to consult https://docs.gunicorn.org/en/stable/settings.html#logging but for now let's keep this file as is and then when someone requests logs we can just add this file as a sink. Cool?

Reply:
You should never store logs in a docker container, the main process is supposed to output to either stdoud or stderr.

Reply:
Ok, will do so!